src/implementations/deploy/token-registry/token-registry.test.ts

```diff
@@ -0,0 +1,75 @@
+import { deployTokenRegistry } from "./token-registry";
+import { join } from "path";
+import { TradeTrustERC721Factory } from "@govtechsg/token-registry";
+import { Wallet } from "ethers";
+
+jest.mock("@govtechsg/token-registry");
+
+const deployParams = {
+  registryName: "Test",
+  registrySymbol: "Tst",
+  network: "ropsten",
+  key: "0000000000000000000000000000000000000000000000000000000000000001"
+};
+
+describe("token-registry", () => {
+  describe("deployTokenRegistry", () => {
```

Comment:
overall: I'm less and less fond of mocked test (I feel this for instance test nothing), mocks are simple so I'm fine (also is it a problem if the test really deploy ?)

missing test I think:
- if there is no key
- if network is incorrect

Reply:
Added test on key, no way to detect if network is incorrect, the deploy will just throw if there is problem with the key. That is covered by the error bubbling test.